shared/actions/chat/inbox.js

```diff
@@ -111,13 +117,14 @@ function * onLoadInbox (): SagaGenerator<any, any> {
       }
       // find it
     } else if (incoming.chatInboxFailed) {
-      console.log('ignoring chatInboxFailed', incoming.chatInboxFailed)
+      console.log('chatInboxFailed', incoming.chatInboxFailed)
       requestIdleCallback(() => {
         incoming.chatInboxFailed.response.result()
       }, {timeout: 100})
 
       yield call(delay, 1)
       const error = incoming.chatInboxFailed.params.error
+      _inboxError = error
```

Comment:
this doesn't mean the entire thing errored, just one row. it shouldn't be involved in the reloading